bin/lib/fastly-extended.js

```diff
@@ -0,0 +1,133 @@
+var Fastly = require('fastly');
```

Comment:
Happy to merge all of this stuff into the upstream [fastly](https://github.com/thisandagain/fastly) module.